core/install_scss.js

```diff
@@ -0,0 +1,10 @@
+var Download = require('download');
```

Comment:
Will this run automatically when we call `npm install` or does something else need to happen first?